gateware/adat_usb2_audio_interface.py

```diff
@@ -405,9 +414,50 @@ def elaborate(self, platform):
             dac2_extractor.selected_channel_in.eq(Mux(usb1_no_channels == 2, 0, 2)),
         ]
 
-        self.wire_up_dac(m, usb1_to_channel_stream, dac1_extractor, dac1, lrclk, dac1_pads)
+        if self.USE_CONVOLUTION:
+            enable_convolver = Signal()
+
+            # load the IR data
+            with wave.open('IRs/DT990_crossfeed_4800taps.wav', 'rb') as wav:
+                ir_data = wav.readframes(wav.getnframes())
+                ir_sample_rate = wav.getframerate()
+
+            ir_sig = np.zeros((len(ir_data) // 6, 2), dtype='int32')
```

Comment:
This part was hardcoded to import PCM_24 WAV files. 
If you prefer a more generic import I prepared the TestPCMImport branch which I can simply merge: https://github.com/fritzbauer/adat-usb2-audio-interface/blob/TestPCMImport/gateware/pcm_import.py#L6

It moves the WAV import to a dedicated file and normalizes any pcm input to the desired bitwidth, regardless whether it is 16,24 or 32 bit.